src/main/drivers/flash_w25n.c

```
@@ -0,0 +1,1122 @@
+    else if (fdevice->io.mode == FLASHIO_QUADSPI) {
+        QUADSPI_TypeDef *quadSpi = fdevice->io.handle.quadSpi;
+
+        // Note: Using HAL QuadSPI there doesn't appear to be a way to send 2 bytes, then blocks of 4 bytes, while keeping the CS line LOW
+        // thus, we have to read the entire BBLUT in one go and process the result.
+
+        uint8_t bblutBuffer[W25N_BBLUT_TABLE_ENTRY_COUNT * W25N_BBLUT_TABLE_ENTRY_SIZE];
+        quadSpiReceive1LINE(quadSpi, W25N_INSTRUCTION_READ_BBM_LUT, 8, bblutBuffer, sizeof(bblutBuffer));
+
+        for (int i = 0, offset = 0 ; i < lutsize ; i++, offset += 4) {
+            if (i < W25N_BBLUT_TABLE_ENTRY_COUNT) {
+                bblut[i].pba = (in[offset + 0] << 16)|in[offset + 1];
```

Cppcheck (2.20.0)

[error] Uninitialized variable

(legacyUninitvar)